cypress/support/pages/CommonElements.ts

```diff
@@ -0,0 +1,49 @@
+import { trim } from "lodash";
+
+export default class CommonElements {
+  parentSelector;
+  primaryBtn;
+  secondaryBtn;
+  secondaryBtnLink;
+  dropdownMenuItem;
+  selectMenuItem;
+  dropdownToggleButton;
+  dropdownSelectToggleButton;
+  dropdownSelectToggleItem;
+
+  constructor(parentSelector: string = "") {
```

Comment:
The type here can be inferred from the default value.
```suggestion
  constructor(parentSelector = "") {
```